dev/io.openliberty.org.crac.1.4/src/org/crac/LibertyCheckpoint.java

```diff
@@ -70,8 +75,9 @@ private void callHooks(Iterator<CheckpointHook> toCall, Consumer<CheckpointHook>
         }
     }
 
-    static public void register(Resource r, Context<? extends Resource> c) {
+    static void register(Resource r, Context<? extends Resource> c) {
         debug(tc, () -> "Registering resource " + r + " from context " + c);
+        registerHookService();
```

Comment:
FYI, this will register the CRAC CheckpointHook service even though CheckpointPhase is INACTIVE.

Reply:
I will take care of this in #28437